### PR TITLE
16 Level Enhancements and add obstacleWall model

### PR DESCRIPTION
**Pull request by Eacaw**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,7 @@
 # MemoryCaptures can get excessive in size.
 # They also could contain extremely sensitive data
 /[Mm]emoryCaptures/
+/[Ww]ebGL Builds/
```
Comment (mmont429) [inline .gitignore:17]: What you got in here?

Comment (Eacaw) [reply .gitignore:17]: It's just the build files I got yesterday

Comment (Eacaw) [reply .gitignore:17]: Can walk you through how to sort build and run it on WebGL at some point

File: Assets/Prefabs/Aesthetic Prefabs/OpenDoorwayPrefab.prefab [hunk 2]
```diff
@@ -0,0 +1,277 @@
+%YAML 1.1
```
Comment (mmont429) [inline Assets/Prefabs/Aesthetic Prefabs/OpenDoorwayPrefab.prefab:1]: Is this just the doorway without the door? or is it something new

Comment (Eacaw) [reply Assets/Prefabs/Aesthetic Prefabs/OpenDoorwayPrefab.prefab:1]: Yep exactly that. Same model, no door, and added lights on either side

File: Assets/Scripts/PlayerMovement.cs [hunk 3]
```diff
@@ -36,7 +39,35 @@ void Update()
             rb.velocity = Vector3.zero;
         }
 
-        transform.Translate(new Vector3(horizontal, 0, vertical) * (speed * Time.deltaTime));
+        // THIS BIT IS THE PROBLEM
```
Comment (mmont429) [inline Assets/Scripts/PlayerMovement.cs:42]: Curious what this does

File: Assets/Scripts/PlayerMovement.cs [hunk 4]
```diff
@@ -36,7 +39,35 @@ void Update()
...
+        // {
+        //     transform.rotation = Quaternion.RotateTowards(transform.rotation, 
+        //     Quaternion.LookRotation(new Vector3(transform.GetComponent<Rigidbody>().velocity.x, 0, transform.GetComponent<Rigidbody>().velocity.z)), 
+        //     Time.deltaTime * RotateSpeed);
+        // }
+
+        transform.Translate(new Vector3(-1 * vertical, 0, horizontal) * (speed * Time.deltaTime));
+    }
+
+    void FixedUpdate()
```
Comment (mmont429) [inline Assets/Scripts/PlayerMovement.cs:53]: What this?

**Review by mmont429 — APPROVED** [review]

File: Assets/Scripts/PlayerMovement.cs (hunk 4 above)
Comment (Eacaw) [reply Assets/Scripts/PlayerMovement.cs:53]: That's the raycast solution to the wall problem

the method name:

void update() happens once per frame
void fixedUpdate() happens once per delta time, so if you're running at 1000fps this doesn't update a million times, and happens in time with the physics engine

File: Assets/Scripts/PlayerMovement.cs (hunk 3 above)
Comment (Eacaw) [reply Assets/Scripts/PlayerMovement.cs:42]: That's the problem I mentioned earlier

File: .gitignore (hunk 1 above)
Comment (mmont429) [reply .gitignore:17]: Ahh make sense, ya maybe when we get closer to the end

**Comment by mmont429** [comment]
WHERE'S THE README UPDATE HUH??

File: Assets/Scripts/PlayerMovement.cs (hunk 4 above)
Comment (mmont429) [reply Assets/Scripts/PlayerMovement.cs:53]: ooooo that's cool

File: Assets/Scripts/PlayerMovement.cs (hunk 3 above)
Comment (mmont429) [reply Assets/Scripts/PlayerMovement.cs:42]: ya but I wonder how borked it is

**Pull request merged**